Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -348,6 +411,101 @@ static void WebAVPlayerLayerView_dealloc(id aSelf, SEL)
     return (WebAVPlayerLayerView *)[theClass alloc];
 }
 
+#if HAVE(PIP_CONTROLLER)
+
+@interface WebAVPictureInPictureContentViewController : AVPictureInPictureContentViewController
+
+- (instancetype)initWithController:(AVPlayerController *)controller;
+@property (nonatomic, nonnull, readonly) AVPlayerController* controller;
+@property (retain) AVPlayerLayer* playerLayer;
+
+@end
+
+static WebAVPictureInPictureContentViewController* WebAVPictureInPictureContentViewController_initWithController(id aSelf, SEL, AVPlayerController* controller)
+{
+    ASSERT(controller);
+
+    WebAVPictureInPictureContentViewController *pipController = aSelf;
+    objc_super superClass { pipController, getAVPictureInPictureContentViewControllerClass() };
+    auto super_init = reinterpret_cast<id(*)(objc_super*, SEL)>(objc_msgSendSuper);
```

Comment:
I guess for this (and the other places below) you don't really need the method name on the variable. But you also can't call it `super` :)

Reply:
Kept this as-is since there is already other (existing) code that uses this naming convention.